runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -829,3 +839,16 @@ TR_J9JITaaSServerSharedCache::addHint(J9Method * method, TR_SharedCacheHint theH
    _stream->write(JITaaS::J9ServerMessageType::SharedCache_addHint, method, theHint);
    _stream->read<JITaaS::Void>();
    }
+
+const void *
+TR_J9JITaaSServerSharedCache::storeSharedData(J9VMThread *vmThread, char *key, J9SharedDataDescriptor *descriptor)
+   {
+   TR_ASSERT(_stream, "stream must be initialized by now");
+   // put well-known class chain offsets in a vector and send it to the client
+   uintptrj_t *classChainOffsets = (uintptrj_t *) descriptor->address;
```

Comment:
Right, this should be more general. I just did it this way because storing array of classChainOffsets is the only case when it gets called currently. I'll change it.